askama_gotham/src/lib.rs

```diff
@@ -1,24 +1,27 @@
 #![deny(elided_lifetimes_in_paths)]
 
 pub use askama::*;
-
 pub use gotham::handler::IntoResponse;
 pub use gotham::state::State;
-pub use hyper::{Body, Response, StatusCode};
+use hyper::header::{HeaderValue, CONTENT_TYPE};
+pub use hyper::{Body, Response};
+
+pub trait TemplateToResponse {
```

Comment:
This change is unrelated and should be in a separate commit (or probably in a separate PR), same for the other integrations.